libraries/tock-tbf/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+//! Tock Binary Format (TBF) header parsing library.
+
+// Parsing the headers does not require any unsafe operations.
+#![forbid(unsafe_code)]
+#![no_std]
+
+pub mod tbfheader;
```

Comment:
This namespaces everything under `tock_tbf::tbfheader`. Was that done to provide for new modules in the future? If not, I think everything should be in the `tock_tbf` namespace rather than a submodule.

Reply:
I didn't think about it, specifically, but doing this was convenient because it meant almost zero changes to process.rs.

I suppose we do need to think about how tock-tbf is organized. I can work on that.